release-note-script/src/main/java/ReleaseNoteCreation.java

```diff
@@ -287,9 +292,9 @@ private List<String> getPullRequestNumbers(String projectId) throws Exception {
       BufferedReader br =
           runSubProcessAndGetOutputAsReader(
               format(
-                  "gh project item-list %s --owner %s --limit 200 | awk -F'\\t' '/%s\\t/ {print"
+                  "gh project item-list %s --owner %s --limit %d | awk -F'\\t' '/%s\\t/ {print"
```

Comment:
FYI: The command seems to internally take care of pagination https://github.com/cli/cli/blob/trunk/pkg/cmd/project/shared/queries/queries.go#L1182-L1186. Looks good.

Reply:
Thank you for the information!
I didn't know the internal mechanism of the command.

Unfortunately, As far as I looked at the [official document](https://cli.github.com/manual/gh_project_item-list) and help command, it seems that `gh project item-list` command can only set the limit of the retrieved items (e.g. PRs) numbers and cannot use the pagination function.

So, I thought I could only set the bigger limit number instead...

Reply:
Ah, sorry for the confusion. I meant the command internally fetches project items with pagination. So, the command won't fetch 10000 project items at once, and your code works safely.